src/modules/45-dashboards/pages/DashboardsCommonHeaderPage.tsx

```diff
@@ -26,15 +26,18 @@ const DashboardsHeader: React.FC = () => {
   const { accountId, folderId } = useParams<{ accountId: string; folderId: string }>()
   const [isOpen, setDrawerOpen] = useState(false)
 
-  const title = breadcrumbs.length ? breadcrumbs[breadcrumbs.length - 1].label : getString('common.dashboards')
+  const title = React.useMemo(
+    () => (breadcrumbs.length ? breadcrumbs[breadcrumbs.length - 1].label : getString('common.dashboards')),
+    [breadcrumbs, getString]
+  )
 
   React.useEffect(() => {
     const titleArray: string[] = [getString('common.dashboards')]
 
     if (breadcrumbs) breadcrumbs.map(breadcrumb => titleArray.push(breadcrumb.label))
```

Comment:
Since you don't care about the return of the `map` function, you should probably use `forEach`
```suggestion
    if (breadcrumbs) {
      breadcrumbs.forEach(({ label }) => titleArray.push(label))
    }
```